### ISSUE TITLE
Pica Characters hex AE and B0

### ISSUE THREAD

**Issue by JonathanRowell** [xbib/bibliographic-character-sets#1]
According to [OCLC/Pica2Unicode](http://support.oclc.org/ggc/richtlijnen/content/pdf/Pica_karakterset_in_Unicode.pdfl) and [OCLC Tabel 7](http://support.oclc.org/ggc/richtlijnen/php/showPresentation.php?id=13&ln=en&par=p-tab-7) these two characters called Alif and Ayn are mapped to 0x02BE and 0x02BF and not 0x02BC and 0x02BB respectively.

Is this an error on your part or are there other considerations to be taken into account.

I'm producing a Javascript implementation and have used the above two documents instead of your implementation as reference.

There are other inconsistancies. At code position octal 316 (0x00CE) in the second document called "Reaction" which is not referenced in the first. "Fleche"  at 307 is also not refered in the same. In the first document there are mention of "Nederlands gulden-teken" and of "Rechtse pijl/Links rechts pijl" with mappings to Unicode but are referenced nowhere else. Can you shed any light on these as well?

**Comment by jprante** [comment]
A long time ago, 20 years or so, I had to map Pica characters for printing. The result is the mapping in this character set project. I found an old document very similar to http://support.oclc.org/ggc/richtlijnen/php/showPresentation.php?id=13&ln=en&par=p-tab-7

There were no glyphs in the target character set for U+02BE/U+02BF (half circles), so I substituted them by apostrophe/turned comma (U+02BB/U+02BC) which look very close. They are only used in arabic transliterations.

"Rechtse pijl/Links rechts pijl" (U+21C4) is also a glyph similar to "GREATER-THAN OR LESS-THAN", the character is used in chemistry for a reaction (U+2277). This is also a question of glyph availability.

"Nederlands gulden-teken" is the old currency symbol of the Netherlands. It was dropped in 2002. Now there is "rightwards arrow" (U+2192), or "Fleche", which means the same but in french.
